.ci/runAdminUiChecks.sh

```diff
@@ -0,0 +1,11 @@
+set -e
+set -x
+
+INITIAL_DIR=$pwd
```

Comment:
I forgot the parantheses:

```suggestion
INITIAL_DIR=$(pwd)
```